src/main/scala/vectorpipe/osm/internal/PlanetHistory.scala

```diff
@@ -148,7 +148,7 @@ private[vectorpipe] object PlanetHistory {
     slices: List[(ElementMeta, Map[Long, Node])]
   ): List[Feature[G, ElementMeta]] = {
     slices.foldLeft(Nil: List[Feature[G, ElementMeta]]) { case (acc, (m, ns)) =>
-      /* If a Node were deleted that the Way still expected, then no Line
```

Comment:
`were` is proper use of the subjunctive mood here.

Reply:
You'd think that I'd know that after le subjonctif was the thing I studied most in French class...